src/Globe/TileMesh.js

```diff
@@ -228,30 +227,29 @@ TileMesh.prototype.setBBoxZ = function(min, max) {
     }
 };
 
-TileMesh.prototype.setTexturesLayer = function(textures, idLayer, slotOffset) {
+TileMesh.prototype.setTexturesLayer = function(textures, layerType, layer) {
     if (this.material === null) {
         return;
     }
     if (textures) {
-        this.material.setTexturesLayer(textures, idLayer, slotOffset);
+        this.material.setTexturesLayer(textures, layerType, layer);
     }
     this.loadingCheck();
 };
 
 TileMesh.prototype.downScaledColorSlot = function(slot) {
     var mat = this.materials[RendererConstant.FINAL];
 
-    return mat.getLevelLayerColor(l_COLOR, slot) < this.level;
-}
+    return mat.Textures[l_COLOR][slot].level < this.level;
```

Comment:
Didn't you intend to hide the `slot` stuff inside LayeredMaterial?
Maybe the `downScaledLayer` function should live in LayeredMaterial?